Core/src/SplineInterpretation/SplineInterpretationElement.cpp

```diff
@@ -128,6 +131,9 @@ void OpenInfraPlatform::Core::SplineInterpretation::SplineInterpretationElement:
 double OpenInfraPlatform::Core::SplineInterpretation::SplineInterpretationElement::getAngle() const noexcept(true) {
 	return angle_; }
 
+double OpenInfraPlatform::Core::SplineInterpretation::SplineInterpretationElement::getAngleDegree() const noexcept(true) {
+	return angle_ * 180.0 / M_PI; }
+

```

Comment:
```suggestion
double OpenInfraPlatform::Core::SplineInterpretation::SplineInterpretationElement::getAngle( bool inDegrees = false ) const noexcept(true) {
	return angle_ * (inDegrees ? 180.0 / M_PI : 1.) ; }
```

Function with a default parameter could be considered. (syntax may be wrong, but I believe you get the idea)

Reply:
Thanks for that suggestion! The concept of default parameter was new to me.
Done in commit d8bb82e.